js/renovation/ui/scroll_view/__tests__/scrollable.test.tsx

```diff
@@ -1039,11 +1057,19 @@ jest.mock('../../../../core/utils/scroll_rtl_behavior', () => () => testBehavior
     describe('Logic', () => {
       describe('Getters', () => {
         describe('cssClasses', () => {
-          it('should add scrolling classes by default', () => {
-            const { cssClasses } = new Scrollable({});
-            expect(cssClasses).toEqual(expect.stringMatching('dx-scrollable'));
-            expect(cssClasses).toEqual(expect.stringMatching('dx-scrollable-native'));
-            expect(cssClasses).toEqual(expect.stringMatching('dx-scrollable-native-generic'));
+          ['desktop', 'ios', 'android'].forEach((platform) => {
+            it('should add scrolling classes by default', () => {
+              (devices.current as Mock).mockImplementation(() => ({ platform }));
```

Comment:
Are you sure we need Mock here?
Does 'devices.current = () => ({ platform })' meet our needs?

Reply:
Fixed